### PR TITLE
Fargate Serverless Workers for Census Data Enrichment and Tile Generation

### PR DESCRIPTION
**Pull request by lscharen**
This PR contains the work done by the GeoPlatform Team to support the prototyping work done between Mike Giddens and @NatHillardUSDS.

Examples of how to deploy and execute the lambda/fargate infrastructure is described in the `infrastructure/README.md` file.

### CODE REVIEW

File: infrastructure/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+The `package.json` file incluses several examples to run against the local source code.  The actual 
+tasks will execute within AWS, so an `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` must be set in
+the `test.env` file.
+
+```bash
+$ cd ./functions/detect-changes-for-worker
+$ npm run test:gdal
+```
+
```
Comment (ginabeena) [inline infrastructure/README.md:29]: Going to be running this local tonight and then merging tonight.

File: infrastructure/functions/detect-changes-for-worker/test.env [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+# Names of the container and task names.
+GDAL_TASK_DEFINITION=sit-justice40-data-harvester-gdal
+GDAL_CONTAINER_DEFINITION=sit-justice40-data-harvester-osgeo-gdal
+TIPPECANOE_TASK_DEFINITION=sit-justice40-data-harvester-tippecanoe
+TIPPECANOE_CONTAINER_DEFINITION=sit-justice40-data-harvester-mgiddens-tippecanoe
+
+# AWS Credentials
+AWS_ACCESS_KEY_ID=
+AWS_SECRET_ACCESS_KEY=
+AWS_DEFAULT_REGION=us-east-1
```
Comment (ginabeena) [inline infrastructure/functions/detect-changes-for-worker/test.env:19]: Pretty sure this is the only dependency we need to add here is the creds.

File: infrastructure/functions/detect-changes-for-worker/ecs.js [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+            containerOverrides: [
+                {
+                    name: containerDefinitionName,
+                    command: containerCommand
+                }
+            ]
+        }
+    };
+
+    logger.info(`Executing ECS Task...`, JSON.stringify(params, null, 2));
```
Comment (ginabeena) [inline infrastructure/functions/detect-changes-for-worker/ecs.js:176]: Where does this end up logging to? Cloudwatch?

**Review by ginabeena — APPROVED** [review]
This looks great and can really probably be merged anytime. I'm going to try hacking on this tonight and building locally, then I'll probably merge. Thanks for your work @lscharen

File: infrastructure/functions/detect-changes-for-worker/ecs.js (hunk 3 above)
Comment (lscharen) [reply infrastructure/functions/detect-changes-for-worker/ecs.js:176]: Yes, it goes into CloudWatch logs under the log group defined [here](https://github.com/usds/justice40-tool/pull/230/files#diff-07622d56543936edef5ffb88b736a04eafb4273a995496b591f0851f144bb8cdR127).

**Pull request merged**